scipy/optimize/_linprog_rs.py

```diff
@@ -23,7 +23,7 @@
 from ._bglu_dense import LU
 from ._bglu_dense import BGLU as BGLU
 from scipy.linalg import LinAlgError
-from numpy.linalg.linalg import LinAlgError as LinAlgError2
+from numpy.linalg import LinAlgError as LinAlgError2
```

Comment:
This looks quite weird. What's the point of importing the same thing twice under different names. I actually checked the file, and the only other use of this seems to be this line (https://github.com/scipy/scipy/pull/14756/files#diff-7be824e870d609afd212fb11b3ee5858537ac96eaa0afe0c447a1f3998315e6eL90):

        except (LinAlgError, LinAlgError2):

Unless there is some other use of it, a better fix would be to clean up the duplicate.

Reply:
SciPy LinalgError is NumPy's LinAlgError anyways. 

https://github.com/scipy/scipy/blob/master/scipy/linalg/misc.py#L1-L6

Reply:
ok then I will clean this! Thank you both.

Reply:
Shall I actually move all usage of `scipy.linalg. LinAlgError` to `numpy.linalg. LinAlgError`?

Reply:
If not, the PR is ready on my side.